backend/handlers/like_handler.py

```diff
@@ -46,7 +46,7 @@ def post(self, user, post_key, comment_id=None, reply_id=None):
         """This method is only meant to give like in post."""
         post = ndb.Key(urlsafe=post_key).get()
         institution = post.institution.get()
-        current_institution_key = self.request.get('currentInstKey')
+        current_institution = self.request.body
```

Comment:
This line doesn't make sense, the body is the structure that encapsulates the date coming from the request. If you're passing the current institution in the body you should create a property current_institution inside the body structure and assign the date to it.